database/migrations/20220407074127-create-whitelist.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("whitelists", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
+      },
+      email: {
+        allowNull: false,
+        unique: true,
+        type: Sequelize.TEXT,
+        validate: {
+          isEmail: true,
```

Comment:
not sure this is going to work - we only validate a suffix, not an email. this change means that organizational emails have to be added individually (eg: for `.edu.sg`, we have to add all indiv emails rather than just the suffix)

Reply:
You're right, thanks for catching! Fixed in https://github.com/isomerpages/isomercms-backend/pull/422/commits/a8805db30268136fa43a14e40d5929b86ce8544b